internal/ui/login/static/templates/login.html

```diff
@@ -42,8 +42,9 @@ <h1>{{t "Login.Title"}}</h1>
 
             {{ $reqid := .AuthReqID}}
             {{range $provider := .IDPProviders}}
-                <a href="{{ externalIDPAuthURL $reqid $provider.IDPConfigID}}" class="button secondary idp-providers">
-                    {{$provider.Name}}
+                <a href="{{ externalIDPAuthURL $reqid $provider.IDPConfigID}}" class="button secondary idp-provider {{idpProviderClass $provider.StylingType}}">
+                    <span class="logo"></span>
+                    <span class="provider-name">{{$provider.Name}}</span>
```

Comment:
what happens if stylingType is empty? did you try as well?

Reply:
yes, than its styled as before